microservices/time-series-analytics/tests/test_classifier_startup.py

```diff
@@ -80,7 +80,7 @@ def test_install_udf_package_runs_pip(monkeypatch, kapacitor_classifier):
     def fake_run(cmd, check=False, **kwargs):
         called.append(cmd)
         class Dummy:
-            pass
+            returncode = 0
```

Comment:
I don't think we need this change, can you cross check?

Rest all changes seems to be fine.

Let's update the commit and PR description as 

"time-series-analytics: Fixed unit tests failure

<description>

Signed-off-by: Name <email id>
"

For PR and commit naming convention, you can refer the Closed tab at https://github.com/open-edge-platform/edge-ai-libraries/issues?q=state%3Aopen%20is%3Apr%20label%3A%222025.2%20Time%20Series%20Analytics%22